README.md

```diff
@@ -93,3 +98,31 @@ client, err := connectors.Salesforce.New(
 ```go
 client, err := connectors.New("salesforce", map[string]any{"workspace": "salesforce-instance-name"})
 ```
+
+## Basic connectors
```

Comment:
Could we name this something like "proxy connectors"? Basic makes it sounds like something barebones which should be copied more or less verbatim, or derived from. Not sure we want to give that impression.

Reply:
Agreed - the terminology isn't great. This concerns product terminology so probably best to wait for @laurenzlong

Reply:
Converting the naming decision into an async call which @laurenzlong will get back to us with